aptos-move/framework/aptos-stdlib/sources/math.move

```diff
@@ -0,0 +1,75 @@
+/// Standard math utilities missing in the Move Language.
+module aptos_std::math {
+
+    /// Return the largest of two numbers.
+    public fun max(a: u64, b: u64): u64 {
```

Comment:
What about `u128`? And even if you don't introduce now, perhaps call these here `max_u64` et. al?

Reply:
Alternatively, we can also put this into the module name, as in `math64::max` et. al

Reply:
I love the way to split the file into the two